src/ReactiveUI.Uno/GlobalSuppressions.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+// This file is used by Code Analysis to maintain SuppressMessage 
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given 
+// a specific target and scoped to a namespace, type, member, etc.
+
```

Comment:
Let's not use a global suppression file, instead move to just attributes above the particular targets.

Eg
```
[SupressMessage("Design", "CA1010:Collections should implement generic interface", Justification = "Deliberate usage")]
public class ReactivePage<T>
```

Reply:
Done.